### ISSUE TITLE
Add the option for user-set hours for Dark Mode

### ISSUE THREAD

**Issue by jmc-figueira** [liri-browser/liri-browser#114]
It has occured to me, from a user perspective, that a user wishing to use dark mode at night only would be limited to central-european/american sunshine hours; a user living, say, in Finland, with only 3 hours of sunlight in the winter, wouldn't benefit from this mode. The starting and ending hours should be user-set.

**Comment by pierremtb** [comment]
@jmc-figueira You are completely right. This is actually a placeholder, because I'm not sure that it's even working like that (I didn't get a chance to test it)

**Comment by VirusKA** [comment]
Yeah good idea.

**Comment by tim-sueberkrueb** [comment]
@pierremtb it works if I'm not mistaken.
